### PR TITLE
Modifications for Digital Input Multiplexers

### LINKED ISSUE #119: Support for HW multiplexer ICs
Multiplexers can be supported in different capacities:
  - Multiplexing of __direct inputs__: mainly __digital inputs__ (similarly to shift registers), but also __analog inputs__
  - __Routing of (existing) I/O blocks__ (extending them with minor modifications):  e.g.  display drivers, digital I/O shift registers.
  
First implementation draft in P.R. #118

### PR DESCRIPTION
**Pull request by DocMoebiuz**
fixes #119

### CODE REVIEW

File: _Boards/Atmel/Board_Uno/MFBoards.h [hunk 1]
```diff
@@ -40,6 +47,8 @@
 #define MAX_ANALOG_INPUTS   6
 #define MAX_OUTPUT_SHIFTERS 2
 #define MAX_INPUT_SHIFTERS  2
+#define MAX_INPUT_SHIFTERS  2
```
Comment (GioCC) [inline _Boards/Atmel/Board_Uno/MFBoards.h:50]: MAX_INPUT_SHIFTERS is repeated

**Review by GioCC — CHANGES_REQUESTED** [review]
Only thing I noticed is a repeated line in [_Boards/Atmel/Board_Uno/MFBoards.h](https://github.com/MobiFlight/MobiFlight-FirmwareSource/pull/169/files#diff-5caba37e39e6c44acb1bbd76bf5e9755b94ece0a9ad28388576181a39e8a8ee9)

File: src/Config.cpp [hunk 2]
```diff
@@ -71,7 +70,7 @@ void          _activateConfig();
 // ************************************************************
 // configBuffer handling
 // ************************************************************
-// reads the EEPROM until NUL terminator and returns the number of characters incl. terminator, starting from given address
+// reads the EEPRROM until NULL termination and returns the number of characters incl. NULL termination, starting from given address
```
Comment (GioCC) [inline src/Config.cpp:73]: Not really worth mentioning, but "EEPRROM" is spelled wrong, and the "\0" char is a NUL char used as terminator; NULL is something that has "no" value (or the "placeholder" value representing that).

**Review by GioCC — APPROVED** [review]

**Review by GioCC — APPROVED** [review]

**Review by GioCC — APPROVED** [review]

**Review by MobiFlight-Admin — APPROVED** [review]
Thanks!

**Pull request merged**